import_export/utils.py

```diff
@@ -23,3 +27,61 @@ def __enter__(self):
     def __exit__(self, *args):
         if self.using_transactions:
             self.context_manager.__exit__(*args)
+
+
+def diff_lines_to_words(text1, text2):
+    # Implemented following the docs:
+    # https://github.com/google/diff-match-patch/wiki/Line-or-Word-Diffs
+    # Only one line is different from diff_linesToChars.
+    # line_end = text.find('\n', lineStart)
+    # --->
+    # line_end = text.find(' ', lineStart)
+    # Variable names are also changed to snake_case for code integrity.
+
+    line_array = []
```

Comment:
What do the `line_array` and `line_hash` do exactly?

Reply:
As I said before, it is some code from diff_match_patch source code and since this package is rewritten in many languages, I think they have copied logic from their cpp source code. It is not pythonic at all. Maybe it is more machine friendly. And I don't know why instead of writing this code in their package, they advocate users to copy and manipulate their code. Weird Google.